protos/schedules.proto

```diff
@@ -2,12 +2,36 @@ syntax="proto3";
 
 package dataform.schedules;
 
+
+message EmailNotificationChannel {
+  repeated string emails = 1;
+}
+
+message SlackNotificationChannel {
+  string webhook_url = 1;
+}
+
+message NotificationChannel {
+ string name = 1;
+ oneof channel {
+    EmailNotificationChannel email_notification = 1;
+    SlackNotificationChannel slack_notification = 2;
+  } 	
+}
+
+message ScheduleNotification {
+ repeated environments = 1;
+ repeated string onEvent = 2;
```

Comment:
we should put a comment on this, pointing at the enum of possible values

Reply:
Added an Enum, and added a comment here